src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs

```diff
@@ -496,7 +494,7 @@ public virtual Parameter BuildConstructorParameter(InputParameter operationParam
         public static bool IsIgnoredHeaderParameter(InputParameter operationParameter)
             => operationParameter.Location == RequestLocation.Header && IgnoredRequestHeader.Contains(operationParameter.NameInRequest);
 
-        private Parameter BuildParameter(in InputParameter operationParameter, Type? typeOverride = null)
+        private Parameter BuildParameter(in InputParameter operationParameter, TypeFactory typeFactory, Type? typeOverride = null)
```

Comment:
same here

Reply:
typeFactory is not in use anyway, removed.